src/web/rustdoc.rs

```diff
@@ -155,6 +134,12 @@ pub fn rustdoc_redirector_handler(req: &mut Request) -> IronResult<Response> {
         Some((krate, path)) => (krate.to_string(), Some(path.to_string())),
         None => (crate_name.to_string(), None),
     };
+
+    if let Some(inner_path) = DOC_RUST_LANG_ORG_REDIRECTS.get(crate_name.as_str()) {
```

Comment:
it feels strange to put this into the rustdoc-handler, in theory we could refactor the shared behaviour into separate methods and then have separate handlers. 

But since axum forbids overlapping routes contrary to iron which just takes the first matching route, we would have to merge the handlers anyways to a certain extend. 

So this is OK with me.